dino.icu.yaml

```
@@ -98,6 +98,11 @@ aryanpatel: # by https://github.com/aryanpatel142006
   type: CNAME
   value: cname.vercel-dns.com. # vercel
 
+baguette: # blog
+  ttl: 600
+  type: CNAME
+  value: oui-oui-baguetti.vercel.app
```

Use cname.vercel-dns.com instead.

Also, where's your verification record?
```suggestion
  value: cname.vercel-dns.com.
```

---

whats that? also wouldnt that go to baguette.vercel.app?

---

Add your domain to vercel. It will ask you to add a verification record. You can add it with all the other verification records at the top of the file. For DNS, you should always dns to cname.vercel-dns.com instead of the direct vercel.app link